### ISSUE TITLE
No live link to the demo.

### ISSUE THREAD

**Issue by Twanawebtech** [IBM-Blockchain/cp-web#63]
I have reviewed the readME of this demo, and it is an interesting one. 
It would be good to feature this demo on our IBM Bluemix github.io page: https://ibm-bluemix.github.io/ but I need a live link for this demo.

In order to feature this demo, it needs to have a live working link in addition to the deploy to Bluemix button so people can see the demo in action before deploying it.

**Comment by sidrmsh** [comment]
@masterDev1985 Can this be done by running it on a certain bluemix instance (not tied to any "org")?

**Comment by masterDev1985** [comment]
@siddarthramesh I've investigated doing this in Bluemix.  At the end of the day, I decided that this isn't something I want to be on the hook for supporting and keeping up 24/7.  If we do it for this demo, then we will probably have to do it for the other demos as well.  Depending on what we do for fabric v1.0, there might be a demo that we keep running.  For now, we aren't supporting a live cp-web demo.
